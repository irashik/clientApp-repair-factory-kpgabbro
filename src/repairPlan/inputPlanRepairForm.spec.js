import React from 'react';
import { shallow, render } from 'enzyme';

import InputPlanRepairForm from './inputPlanRepairForm';



describe('<InputPlanRepairForm />', () => {

    test('snapShot correctly', () => {

        const component = shallow(<InputPlanRepairForm />);
        expect(component).toMatchSnapshot();

    });
});