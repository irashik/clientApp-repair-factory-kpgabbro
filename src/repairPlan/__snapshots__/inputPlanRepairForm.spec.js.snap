// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputPlanRepairForm /> snapShot correctly 1`] = `
<Modal
  animation={false}
  aria-labelledby="contained-modal-title-vcenter"
  autoFocus={true}
  backdrop="static"
  dialogAs={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "render": [Function],
    }
  }
  dialogClassName="modal-90w"
  enforceFocus={true}
  key=""
  keyboard={true}
  restoreFocus={true}
  show={false}
  size="lg"
>
  <ModalHeader
    closeButton={false}
    closeLabel="Close"
  >
    <ModalTitle
      id="contained-modal-title-vcenter"
    >
      Форма ввода данных о плановых работах
    </ModalTitle>
    <button
      aria-label="Close"
      className="btn-close"
      onClick={[Function]}
      type="button"
    />
  </ModalHeader>
  <ModalBody
    className="show-grid"
  >
    <Container
      fluid={true}
      id="input-module"
    >
      <Row>
        <Col
          md="auto"
        >
          <FormControl
            id="inputEquipment"
            onChange={[Function]}
            placeholder="Выберите оборудование"
            size="sm"
            title=""
            type="text"
            value=""
          />
          <SearchList
            filter=""
            onSelectEquipment={[Function]}
          />
          <br />
          <label>
            Трудозатраты
          </label>
          <FormControl
            id="inputSpendingJob"
            onChange={[Function]}
            pattern="[0-9]*"
            placeholder="трудозатраты"
            size="sm"
            type="number"
            value={0}
          />
        </Col>
        <Col
          md={6}
        >
          <FormInputRepair
            count={
              Array [
                1,
              ]
            }
            onHandleRecordRepair={[Function]}
            onLoadRepair={Array []}
          />
        </Col>
        <Col
          md={2}
        >
          <InputGroupButtonSmall
            onHandleRepairCount={[Function]}
          />
        </Col>
      </Row>
      <Row>
        <label>
          Статус задачи
        </label>
        <FormControl
          aria-label="Выберите статус задачи"
          as="select"
          onChange={[Function]}
          size="sm"
          value="DRAFT"
        >
          <option
            value="DRAFT"
          >
            Заготовка
          </option>
          <option
            value="CANCELLED"
          >
            Отменено
          </option>
          <option
            value="FINISHED"
          >
            Завершено
          </option>
          <option
            value="DEFERRED"
          >
            Отложено
          </option>
          <option
            value="INWORK"
          >
            В работе
          </option>
          <option
            value="ACTIVE"
          >
            Активная
          </option>
        </FormControl>
      </Row>
      <Row>
        <FormControl
          id="inputPriority"
          onChange={[Function]}
          placeholder="Приоритетность задачи"
          size="sm"
          type="text"
          value=""
        />
      </Row>
      <Row>
        <FormControl
          as="textarea"
          id="inputComment"
          onChange={[Function]}
          placeholder="Комментарии"
          rows={3}
          size="sm"
          value=""
        />
      </Row>
      <Row />
      <Row>
        <DateCreatedView
          dateCreated=""
        />
        <DateFinishedView
          dateFinished=""
        />
        <AuthorView
          author=""
        />
      </Row>
    </Container>
  </ModalBody>
  <ModalFooter>
    <BtnView
      onLoadRecord={Array []}
    />
  </ModalFooter>
</Modal>
`;
